src/key.ts

```diff
@@ -1,2 +1,2 @@
-export const api_key = '250b65ac19c1d23b70718726a42ee160'
+export const apiKey = '250b65ac19c1d23b70718726a42ee160'
 export const PROXY_URL = 'https://cors-anywhere.herokuapp.com/'
```

Comment:
This proxy business is interesting to me. It's weird that the app didn't work for Firefox and probably something we'd want to look into at some point. Consider making an Issue to track this work. If you aren't using this variable, you could also remove it. It's better to track issues in an Issue tracker. You can paste in the issue this URL and some notes about how you were able to fix the issue for your future self.

Reply:
I believe this is unnecessary at this point, and it's not being used (don't think it's worth pursuing further). I'll shift the focus more on removing the public key and completely removing this ts file altogether.